### DISCUSSION TITLE
Improvement to the Wiki

### DISCUSSION THREAD

**Discussion by GrampaWildWilly** [aclap-dev/video-downloadhelper#212] [Help and Q&A]
### Your question

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/28ec5f8b-5ec5-4e94-bc36-9ba537d70da9)

### Addon & CoApp version and details

Irrelevant.  This is not an issue with the software but rather an issue with our GitHub repository.

**Comment by jcfrog** [comment]
it's quite logical to me, that's the way wikis work, you can't list all pages, the search field is the main entrance.

**Comment by jcfrog** [comment]
but it's true we miss a simple search in the wiki, for the moment you can only search on pages titles
Something like https://github.com/search?q=repo%3Aaclap-dev%2Fvideo-downloadhelper+CoApp&type=wikis

**Reply by GrampaWildWilly** [reply]
I'm not so sure that's true, Jérôme.  Take a look at #207.  I put some stuff in there that is a candidate for inclusion in the Wiki as a tutorial.  Also, I offer this as an alternative to your search results URL:

https://github.com/search?q=coapp&type=discussions

**Reply by jcfrog** [reply]
yes, there are ways to find info, but for the wiki it misses a simple search box at the top.

**Reply by GrampaWildWilly** [reply]
Hmmm....... There's a search box at the top of the Wiki page.  When you click it, it proposes a search within our repository.  That would include our Wiki.  You can search on things & the results will propose things in our Wiki.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/e5dbfb01-974a-4785-bf97-e9b3fc6f1131)
![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/64b4893c-047b-4aa9-977c-9ddd4d6095a8)

**Reply by jcfrog** [reply]
Yes I know, I'm thinking about the majority of users who have not your geek skills :)

**Reply by GrampaWildWilly** [reply]
That's what tutorials are for. 😄
